api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -430,6 +497,14 @@ public Builder preventClose() {
             return this;
         }
 
+        /**
+         * Toggles compatibility with Geyser software
+         */
+        public Builder geyserCompat(boolean enabled) {
```

Comment:
as you mentioned `disableGeyserCompat` would make more sense considering its enabled by default